server/sonar-server/src/main/resources/org/sonar/server/design/ws/example-dsm.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
this example deserves at least two or three components, else it's hard to understand the concept